awslimitchecker/connectable.py

```diff
@@ -86,7 +86,14 @@ def _boto3_connection_kwargs(self):
         :rtype: dict
         """
         kwargs = {'region_name': self.region}
-        if self.account_id is not None:
+        if self.profile_name is not None:
```

Comment:
I'm a bit confused by these next few lines... if we're using a profile in one of the config files that Boto knows about, shouldn't we just have to pass in the profile name? I'm not clear on why we're reading the credentials instead of just setting ``kwargs['profile_name'] = self.profile_name``